docs/schema/association-element-in-associations-subscription-schema.md

```diff
@@ -115,15 +116,15 @@ The following sections describe attributes, child elements, and parent elements.
 </thead>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="localizeddisplaynames-element-in-metadataobject-subscription-schema.md">LocalizedDisplayNames element in MetadataObject (Subscription Schema)</a></span></p></td>
+<td align="left"><p><span><a href="localizeddisplaynames-element-in-metadataobject-subscription-schema.md">LocalizedDisplayNames element in MetadataObject (Subscription Schema)</a></span></p></td>
```

Comment:
If this page is being updated:
- don't just remove the attribute, but remove the entire span
- but, it should be converted from an HTML table => Markdown table